lib/shopping_cart.rb

```diff
@@ -0,0 +1,32 @@
+class ShoppingCart
+  attr_reader :name, :capacity, :products
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
```

Comment:
We can convert this string into an integer here. So we should have
`@capacity = capacity.to_i`